harvard-lts/Harvard-Geospatial-Library
# Adding curl and libcurl4.

**Adding curl and libcurl4.**
* * *


# What does this Pull Request do?
This adds curl and libcurl4 to the Dockerfiles so that the new version of GeoBlacklight builds properly. CURIOSity has curl as well, so this shouldn't pose an issue.


# How should this be tested?

A description of what steps someone could take to:
* Rebuild the container using the `add-curl` branch
* Confirm that HGL builds and works as-usual

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz
